### PR TITLE
upgrade SDK to fix tagging issue

### PR DESCRIPTION
**Pull request by erkangz**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -7,8 +7,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [10.x, 12.x]
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        node-version: [12.x]
+        os: [ubuntu-latest]
```
Comment (austin-rausch) [inline .github/workflows/build.yml:11]: I thought we wanted to keep these multi platform for contract developers who might be on windows.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jupiterone/graph-cisco-meraki",
-  "version": "1.2.0",
+  "version": "1.2.2",
```
Comment (austin-rausch) [inline package.json:3]: Did something happen to 1.2.1?

File: .github/workflows/build.yml (hunk 1 above)
Comment (erkangz) [reply .github/workflows/build.yml:11]: ah ok, gtk. this one shouldn't matter.

File: package.json (hunk 2 above)
Comment (erkangz) [reply package.json:3]: i just needed to re-tag :)

**Pull request merged**